VSRAD.Package/BuildTools/Errors/Parser.cs

```diff
@@ -20,11 +20,20 @@ public static ICollection<Message> ParseStderr(IEnumerable<string> outputs)
                     string line;
                     while ((line = reader.ReadLine()) != null)
                     {
-                        var message = ParseScriptMessage(line) ?? ParseKeywordMessage(line) ?? ParseClangMessage(line);
-                        if (message != null)
+                        var message = ParseKeywordMessage(line) ?? ParseClangMessage(line);
+                        var scriptMessages = ParseScriptMessage(line);
+                        if (message != null || scriptMessages != null)
```

Comment:
This condition is not necessary since  `message != null` and `scriptMessages != null` are handled differently.